draft-ietf-quic-multipath.md

```diff
@@ -358,16 +358,20 @@ on different 4-tuples due to, e.g., NAT rebinding. In such a case, the receiver
 as specified in {{Section 9.3 of QUIC-TRANSPORT}} by initiating path validation
 but MUST use a new connection ID for the same Path ID.
 
-This proposal adds five multipath control frames for path management:
+This proposal adds six multipath control frames for path management:
 
 - PATH_ABANDON frame for the receiver side to abandon the path
 (see {{path-abandon-frame}}),
 - PATH_BACKUP and PATH_AVAILABLE frames to express a preference
 in path usage (see {{path-backup-available-frame}}), and
 - MAX_PATH_ID frame (see {{max-paths-frame}}) for increasing the limit of
-path identifiers, and PATHS_BLOCKED frame (see {{paths-blocked-frame}})
+path identifiers,
+- PATHS_BLOCKED frame (see {{paths-and-cids-blocked-frame}})
```

Comment:
Editorial suggestion to combine these to one bullet point again (total 3 proposed commits): 
```suggestion
- PATHS_BLOCKED and PATH_CID_BLOCKED frames (see {{paths-and-cids-blocked-frame}})
```